antlr/src/main/antlr/FSH.g4

```diff
@@ -91,7 +113,7 @@ COMMA:              ',';
 // PATTERNS
 
                  //  "    CHARS    "
-STRING:             '"' (~[\\"])* '"';
+STRING:             '"' (~[\\"] | '\\"' | '\\\\')* '"';
```

Comment:
Since you changed the existing `STRING` rule here, can you update the importer to accommodate the change?  Basically, anywhere that parses a rule that uses the `STRING` token, I think it will need to:
* unescape any `\"`
* unescape any `\\`